infrastructure/service-bus.tf

```diff
@@ -33,3 +33,15 @@ resource "azurerm_key_vault_secret" "sscs-hmc-servicebus-hared-access-key" {
   value        = data.azurerm_key_vault_secret.hmc-servicebus-shared-access-key.value
   key_vault_id = data.azurerm_key_vault.sscs_key_vault.id
 }
+
+resource "azurerm_servicebus_subscription_rule" "servicebus_subscription_rule_sscs" {
+  name            = "hmc-servicebus-subscription-rule=bba3"
+  subscription_id = azurerm_servicebus_subscription.servicebus_subscription.id
```

Comment:
Error

A managed resource "azurerm_servicebus_subscription" "servicebus_subscription" has not been declared in the root module.